restler/restler_settings.py

```diff
@@ -704,12 +706,54 @@ def allow_grammar_py_user_update(self):
 
     @property
     def token_refresh_cmd(self):
-        return self._token_refresh_cmd.val
+        if self._token_refresh_cmd.val:
+            return self._token_refresh_cmd.val
+        elif 'token' in self._authentication_settings.val and 'cmd' in self._authentication_settings.val['token']:
+            return self._authentication_settings.val['token']['cmd']
+        else:
+            return None
 
     @property
     def token_refresh_interval(self):
-        return self._token_refresh_interval.val
+        if self._token_refresh_interval.val:
+            return self._token_refresh_interval.val
+        elif 'token' in self._authentication_settings.val and 'token_refresh_interval' in self._authentication_settings.val['token']:
+            return self._authentication_settings.val['token']['token_refresh_interval']
+        else:
+            return None
+    @property
+    def token_location(self):
+        if 'token' in self._authentication_settings.val and 'location' in self._authentication_settings.val['token']:
+            return self._authentication_settings.val['token']['location']
+        else:
+            return None
+    @property
+    def token_module_file(self):
+        if 'token' in self._authentication_settings.val and 'module' in self._authentication_settings.val['token'] and 'file' in self._authentication_settings.val['token']['module']:
+            return self._authentication_settings.val['token']['module']['file']
+        else:
+            return None
+    @property
+    def token_module_method(self):
```

Comment:
>token_module_method [](http://example.com/codeflow?start=8&length=19)

I believe this should be 'token_module_function' given standard python terminology

Reply:
changing

Reply:
Going to rename this everywhere